### PR TITLE
[zh-cn] sync resource-bin-packing create-cluster-kubeadm encrypt-data

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/scheduling-eviction/resource-bin-packing.md
content/zh-cn/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md
content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
This file has some more in-synced content:

```
content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md
```
Pls confirm it by running `./scripts/lsync.sh` on your `main` branch.

**Comment by my-git9** [comment]
> This file has some more in-synced content:
> 
> ```
> content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md
> ```
> 
> Pls confirm it by running `./scripts/lsync.sh` on your `main` branch.

Updated

**Comment by windsonsea** [comment]
-_ ! #44532 merged last night. Sorry for late review.

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 1]
```diff
@@ -516,7 +512,149 @@ In the KMS case, an attacker who intends to get unauthorised access to the plain
...
+```shell
+head -c 32 /dev/urandom | base64
+```
+
+<!--
+You can use `/dev/hwrng` instead of `/dev/urandom` if you want to
+use your PC's built-in hardware entropy source. Not all Linux
+devices provide a hardware random generator.
+-->
+如果你想使用 PC 的内置硬件熵源，可以使用 `/dev/hwrng` 而不是 `/dev/urandom`。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:601]: ？

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 2]
```diff
@@ -516,7 +512,149 @@ In the KMS case, an attacker who intends to get unauthorised access to the plain
...
+并非所有 Linux 设备都提供硬件随机生成器。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:602]:
```suggestion
并非所有 Linux 设备都提供硬件随机数生成器。
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 3]
```diff
@@ -516,7 +512,149 @@ In the KMS case, an attacker who intends to get unauthorised access to the plain
...
+{{% /tab %}}
+{{< /tabs >}}
+
+
+{{< note >}}
+<!--
+Keep the encryption key confidential, including whilst you generate it and
+ideally even after you are no longer actively using it.
+-->
+保持加密密钥的机密性，包括在生成密钥时，甚至在你不再主动使用密钥后也是理想的情况。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:639]:
```suggestion
保持加密密钥的机密性，包括在生成密钥时，甚至理想的情况下在你不再主动使用密钥后也要保密。
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 4]
```diff
@@ -823,72 +959,155 @@ or script an update.
...
+
+When you have configured encryption at rest for an API (for example: the API kind
+`Secret`, representing `secrets` resources in the core API group), you **must** ensure
+that all those resources in this cluster really are encrypted at rest. Check this before
+you carry on with the next steps.
+-->
+
+此更改可防止 API 服务器检索标记为静态加密但实际上以明文形式存储的资源。
+
+当你为 API 配置静态加密时（例如：API 种类 `Secret`，代表核心 API 组中的 `secrets` 资源），
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:986]:
```suggestion
当你为某个 API 配置静态加密时（例如：API 种类 `Secret`，代表核心 API 组中的 `secrets` 资源），
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 5]
```diff
@@ -823,72 +959,155 @@ or script an update.
...
+你**必须**确保该集群中的所有这些资源确实在静态时进行了加密，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:987]:
```suggestion
你**必须**确保该集群中的所有这些资源确实被静态加密，
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 6]
```diff
@@ -823,72 +959,155 @@ or script an update.
...
+在不发生停机的情况下更改 Kubernetes 的加密密钥需要多步操作，特别是在有多个 `kube-apiserver`
 进程正在运行的高可用环境中。
 
-1. 生成一个新密钥并将其添加为所有服务器上当前提供程序的第二个密钥条目
-1. 重新启动所有 `kube-apiserver` 进程以确保每台服务器都可以使用新密钥进行解密
-1. 将新密钥设置为 `keys` 数组中的第一个条目，以便在配置中使用其进行加密
-1. 重新启动所有 `kube-apiserver` 进程以确保每个服务器现在都使用新密钥进行加密
-1. 运行 `kubectl get secrets --all-namespaces -o json | kubectl replace -f -`
+1. 生成一个新密钥并将其添加为所有控制平面节点上当前提供程序的第二个密钥条目
+1. 重新启动所有 `kube-apiserver` 进程以确保每台服务器都可以使用新密钥加密的任何数据
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:1049]:
```suggestion
1. 重新启动所有 `kube-apiserver` 进程以确保每台服务器都可以使用新密钥加密任何数据
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 7]
```diff
@@ -823,72 +959,155 @@ or script an update.
...
+1. 对新的加密密钥进行安全备份。如果你丢失了此密钥的所有副本，则需要删除在丢失的密钥下加密的所有资源，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:1050]:
```suggestion
1. 对新的加密密钥进行安全备份。如果你丢失了此密钥的所有副本，则需要删除用已丢失的密钥加密的所有资源，
```

File: content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md [hunk 8]
```diff
@@ -823,72 +959,155 @@ or script an update.
...
-To disable encryption at rest, place the `identity` provider as the first entry in the config
-and restart all `kube-apiserver` processes.
+To disable encryption at rest, place the `identity` provider as the first
+entry in your encryption configuration file:
 -->
 ## 解密所有数据    {#decrypting-all-data}
 
-要禁用静态加密，请将 `identity` 提供程序
-作为配置中的第一个条目并重新启动所有 `kube-apiserver` 进程。
+此示例演示如何停止静态加密 Secret API。如果你要加密其他 API 类型，请调整对应步骤来适配。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/encrypt-data.md:1069]:
```suggestion
此示例演示如何停止静态加密 Secret API。如果你所加密的是其他 API 类型，请调整对应步骤来适配。
```

**Comment by windsonsea** [comment]
/approve
thanks.

**Pull request merged**